frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -601,7 +607,28 @@ export const surveyLogic = kea<surveyLogicType>([
                     kind: NodeKind.DataTableNode,
                     source: {
                         kind: NodeKind.EventsQuery,
-                        select: ['*', `properties.${surveyResponseProperty}`, 'timestamp', 'person'],
+                        select: [
+                            '*',
+                            ...survey.questions.map((q, i) => {
+                                if (q.type === SurveyQuestionType.MultipleChoice) {
+                                    // Join array items into a string
+                                    return `coalesce(arrayStringConcat(JSONExtractArrayRaw(properties, '${getResponseField(
+                                        i
+                                    )}'), ', ')) -- ${q.question}`
+                                } else if (q.type === SurveyQuestionType.Rating) {
+                                    // Decrement 1 due to 0 - 10 scale
+                                    return `coalesce(JSONExtractInt(properties, '${getResponseField(i)}') - 1) -- ${
```

Comment:
I'm not sure why you're decrementing 1 here, mind explaining? 🤔  The responses should be either between 1-5 or 0-10; depending on the scale.

Reply:
for the visualisations we were treating ratings as strings (I think), might make sense to do the same here, as if there's an invalid response making it in, I'm pretty sure this would break, while JSONExtractString will not break.